src/res/apps.yaml

```diff
@@ -392,4 +392,39 @@ items:
     #twitter group handle
     twitter: https://twitter.com/Africanjob
     image: https://pbs.twimg.com/profile_images/1218827868526071808/bFGDUTns_400x400.jpg
-#--------------------------------------------------------------
+#--------------------------------------------------------------------
+  - title: Appfabrik
+    category: Software Company
+    #max of five tags
+    tags:
+      - Software Solution
+      - Fintech
+    description: |
+      Faisant de l’économie numérique notre principal domaine d’activité, nous sommes une start-up germano-camerounaise spécialisée dans le domaine des technologies innovantes. Né du souci d’allier nos habitudes quotidiennes à l’usage des nouveaux modes de communication nous menons des recherches afin de déceler les besoins et les manques divers auxquels nous proposons des solutions essentiellement technologiques. De la fusion des termes « Application » (App) et « Fabrication » (Fabrik), AppFabrik fait donc en grande partie dans la création d’applications mobiles et de logiciels informatiques. D’où notre slogan « Software solution ». Faisant de la Financial Technologie (FINTECH) notre crédo, nous prônons la simplification des procédures financières par des méthodes essentiellement numériques. Basé à Douala depuis février 2017, la jeune équipe de cette structure caresse l’ambition de participer activement au développement numérique du continent Africain. Ceci à travers la création d’un environnement productif et la promotion des solutions durables adaptées aux différentes conditions de nos cibles. Avec les nombreuses stratégies que nous déployons au quotidien, nous touchons de nombreux particuliers et des structures qui en retour nous témoignent de leur confiance.
```

Comment:
Do you think that this text will fit in the card ? Maybe we should have a text shorter than this.
<img width="443" alt="Screenshot 2021-12-10 at 22 53 36" src="https://user-images.githubusercontent.com/5704817/145646299-d46e29a7-87cf-4d24-b747-2a32c016f865.png">

Reply:
It seems like this is not an application or a community but a company. I am not sure we want to add service company on the website. Let me know what you think

Reply:
You're right, I will delete that